test/lib/skip.py

```diff
@@ -60,3 +60,7 @@
     "select curdate\\(\\)",
     "select current_date\\(\\)",
 ]
+
+skip_files = set([
+    # 'test_parquet_dict_null_predicate'
```

Comment:
remove ?

Reply:
It can be used as an example of skipping some test files.